### DISCUSSION TITLE
Rails World

### DISCUSSION THREAD

**Discussion by manuelfcreis** [kaspth/oaken#52] [General]
Who's at Rails world? Would love to meet you guys in person. Should we have a mini meetup sometime these days?

**Comment by codergeek121** [comment]
I'm at Railsworld currently, would love to meet and discuss 👍

**Comment by kaspth** [comment]
A mini-meetup sounds like a great idea! I think @alexandreruban's there too.

I'm not there, but hope you have a good time

**Comment by alexandreruban** [comment]
Hey!
Yes I'm there too! Should we meet after the last keynote?

**Comment by manuelfcreis** [comment]
Sounds great to me! @codergeek121 @alexandreruban lets meet near the  coffee place with an espresso machine after the keynote today?

**Reply by alexandreruban** [reply]
Perfect for me! :)
